src/averageAges.js

```diff
@@ -20,6 +20,18 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const male = people.filter(person => century
```

Comment:
`male` refers to a single value, and what you get is an array – use `males` instead